guides/Identities.md

```diff
@@ -0,0 +1,77 @@
+# About Identities
+
+Each request to a Wix API is associated with a specific identity type, which determines access permissions. If a [method has identity restrictions](#methods-with-restricted-identities), the method reference will include a note specifying the allowed identity types.
```

Comment:
Suggestion to turn the sentence around: Access permissions are determined by the specific identity type that is ?included in the API call.

Reply:
Does the second sentence relate to the documentation, that is the method will include a note in the documentation ?

Reply:
how about this:
Every request to a Wix API is made by a specific identity type. For each call, the caller's identity is checked and confirmed to have the necessary permissions.

Reply:
yes, the second sentence is about the documentation. copied directly from the SDK article